torchtune/training/__init__.py

```diff
@@ -113,12 +108,7 @@
     "set_torch_num_threads",
     "shard_model",
     "get_shard_conditions",
-    "prepare_model_for_fsdp_with_meta_device",
     "validate_no_params_on_meta_device",
-    "contains_fsdp",
-    "FSDPPolicyType",
-    "get_full_finetune_fsdp_wrap_policy",
-    "lora_fsdp_wrap_policy",
```

Comment:
Same here to be removed from the training API docs.

Reply:
Yeah that would be good to remove as well